### ISSUE TITLE
Metered connection detection is faulty

### ISSUE THREAD

**Issue by Shroopy** [canton7/SyncTrayzor#662]
SyncTrayzor version 1.1.19, SyncThing version 1.18.1
I'm trying to sync a very large connection of files, and occasionally I get the message on one of the computers that the other is using a metered connection and that it paused the transfer, even though neither is using a metered connection.

**Comment by canton7** [comment]
Yeah, I've seen it misbehave occasionally. I'm not sure why Windows is saying what it's saying, but I'm not sure there's much I can do about it. You can disable it in the settings.

**Comment by Shroopy** [comment]
Where in the settings can I disable it? I'm having trouble finding the setting.

**Comment by canton7** [comment]
File -> Settings -> Pause devices which connect over a metered network
